pkg/helmt/helmt.go

```diff
@@ -129,8 +129,18 @@ func template(name string, chart string, values []string, namespace string, skip
 	return execute("helm", execOpts{}, args...)
 }
 
-func fetch(repository, chart, version string) error {
-	return execute("helm", execOpts{}, "fetch", "--repo", repository, "--version", version, chart)
+func fetch(repository, chart, version string, username, password string) error {
+	args := []string{"fetch"}
+	args = append(args, "--repo", repository)
+	args = append(args, "--version", version)
+	if username != "" {
+		args = append(args, "--username", username)
+	}
+	if password != "" {
+		args = append(args, "--password", password)
```

Comment:
Should there be any tying between username and password? I.e., when you pass one without another, should the tool give you a nice feedback?

Reply:
I checked how `helm` reacts if you provide only one of the two parameters. There is no special feedback. So I did not implement additional logic for `helmt` either.